main.go

```diff
@@ -183,6 +185,8 @@ func main() {
 		os.Exit(1)
 	}
 
+	setupChecks(mgr)
```

Comment:
I'd expect this to be configurable for the users and to `9440` if no flag is passed

Reply:
The flag was added in #764 ,but not wired up.
Need to add something along the lines of 
`HealthProbeBindAddress:  healthAddr`
to `ctrl.NewManager`